src/client/java/io/d2a/fuzzy/screens/FuzzyCommandScreen.java

```diff
@@ -98,7 +98,8 @@ protected void init() {
                 resultBoxHeight,
                 resultBoxY
         );
-        resultListWidget.setRenderBackground(false);
+
+        // resultListWidget.setRenderBackground(false); // TODO?
```

Comment:
Not sure whether this needs any further stuff to work, this method does not exist anymore so I've just gone ahead and commented that out for now.

Reply:
Yep, looks to be working fine without this